package.json

```
@@ -48,19 +48,17 @@
     "hexo-fs": "^2.0.0",
     "hexo-i18n": "^1.0.0",
     "hexo-log": "^1.0.0",
+    "hexo-renderer-nunjucks": "github:oncletom/hexo-renderer-nunjucks#master",
```

```suggestion
    "hexo-renderer-nunjucks": "^2.0.0",
```

To be merged once `hexo-renderer-nunjucks` is republished on npm.